alpha/external-web/Dockerfile

```diff
@@ -8,4 +8,20 @@ COPY . /usr/src/app
 
 EXPOSE 3000
 
+# Install PDF Toolkit. Required for PDF compression.
+ENV PDFTK_VERSION 2.02
+
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
```

Comment:
Probably not.  

`apt-get update` updates the list of packages that it can install. `apt-get upgrade` updates the software installed already I.e. whatever is on the container already prior to us added gcj, g++, etc.